src/app/dashboard/plugin-config.html

```diff
@@ -1,13 +1,15 @@
-<md-dialog>
+<md-dialog flex="30">
   <md-dialog-content>
-    <!-- {{vm.plugin.title}} -->
+    <h2>{{vm.plugin.title}}</h2>
```

Comment:
I think this should be the plug-in's original title so you remember what you're adding, like "Hybrid Storage Browser" vs. "Hybrid Storage Browser (2)"